packages/@react-aria/slider/src/useSlider.ts

```diff
@@ -173,8 +173,18 @@ export function useSlider(
       ...fieldProps
     },
     trackProps: mergeProps({
-      onMouseDown(e: React.MouseEvent<HTMLElement>) { onDownTrack(e, undefined, e.clientX, e.clientY); },
-      onPointerDown(e: React.PointerEvent<HTMLElement>) { onDownTrack(e, e.pointerId, e.clientX, e.clientY); },
+      onMouseDown(e: React.MouseEvent<HTMLElement>) {
+        if (e.button !== 0) {
```

Comment:
Looks like Chrome and Safari on macOS show the context menu with Ctrl + click, but `e.button` is still 0, whereas in Firefox `e.button` is 2. Maybe we should ignore if a modifier key is pressed?